app/liquid/views/layouts/master.liquid

```diff
@@ -0,0 +1,49 @@
+<div class="cover-photo" style="background-image: url({{ images[0].urls.large }})">
+
+  <div class="cover-photo__overlay">
+
+    <a href="/">
+      <div class="header-logo header-logo--simple checks-top checks-top--at-top">
+        <div class="header-logo__logo sumofus-logo--negative"></div>
+        <div class="header-logo__tagline sumofus-logo--tagline">Fighting for people over profits</div>
+      </div>
+    </a>
+
+    <div class="cover-photo__titles">
+      <div class="center-content">
+        <div class="center-content__big-column">
+          <!-- one line to be able to check :empty  -->
+          <h3 class="cover-photo__target typography__highlight--dark typography__highlight--simple">{% include 'target' %}</h3>
+          <div class="typography__highlight-wrapper">
+            <h1 class="cover-photo__title typography__highlight">
+              <span>{{ title }}</span>
+            </h1>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+
+<div class="center-content">
+  <div class="center-content__big-column">
+
+    <div class="body-text main-feature">
+      {{ content }}
+
+      {% unless link_list.blank? %}
+        <hr class="stubby-hr" />
+        <h3>More information</h3>
+        {% include 'links' %}
+      {% endunless %}
+    </div>
+  </div>
+  <div class="center-content__small-column">
+    <div class="action-bar action-bar--elevated action-bar--mobile-view--closed">
+      {% include 'action' %}
+    </div>
+  </div>
+</div>
+
+{% include 'simple_footer' %}
```

Comment:
Interesting decision to move liquid view templates into `app/liquid` :+1: 

Can we rename this layout to `sumofus`, and keep the original `master`. Having the generic, does everything, template is important, I think. You could also rename it: `master -> default`.